Android.mk missing update for double precision?

[Updating](https://github.com/libpd/pd-for-android/pull/122) pd-for-android to the latest commit is failing to build:
```
In file included from /home/runner/work/pd-for-android/pd-for-android/PdCore/src/main/jni/libpd/libpd_wrapper/util/z_queued.c:11:
/home/runner/work/pd-for-android/pd-for-android/PdCore/src/main/jni/libpd/libpd_wrapper/util/z_queued.h:43:47: error: unknown type name 't_libpd_doublehook'
EXTERN void libpd_set_queued_doublehook(const t_libpd_doublehook hook);
```

Any clues @danomatika ?

I don’t use Android, so no idea. The android makefile may need to be updated?

enohp ym morf tnes
-----------
Dan Wilcox
danomatika.com
robotcowboy.com


> On May 18, 2022, at 5:55 AM, Joe Bowbeer ***@***.***> wrote:
> 
> ﻿
> Updating pd-for-android to the latest commit is failing to build:
> 
> In file included from /home/runner/work/pd-for-android/pd-for-android/PdCore/src/main/jni/libpd/libpd_wrapper/util/z_queued.c:11:
> /home/runner/work/pd-for-android/pd-for-android/PdCore/src/main/jni/libpd/libpd_wrapper/util/z_queued.h:43:47: error: unknown type name 't_libpd_doublehook'
> EXTERN void libpd_set_queued_doublehook(const t_libpd_doublehook hook);
> Any clues @danomatika ?
> 
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you were mentioned.

--- a/Android.mk
+++ b/Android.mk
@@ -39,8 +39,8 @@ PD_SRC_FILES := \
   libpd_wrapper/x_libpdreceive.c libpd_wrapper/z_libpd.c \
   libpd_wrapper/util/ringbuffer.c libpd_wrapper/util/z_queued.c \
   libpd_wrapper/z_hooks.c libpd_wrapper/util/z_print_util.c
-PD_C_INCLUDES := $(LOCAL_PATH)/pure-data/src $(LOCAL_PATH)/libpd_wrapper \
-  $(LOCAL_PATH)/libpd_wrapper/util
+PD_C_INCLUDES := $(LOCAL_PATH)/libpd_wrapper $(LOCAL_PATH)/libpd_wrapper/util \
+  $(LOCAL_PATH)/pure-data/src
 PD_CFLAGS := -DPD -DHAVE_UNISTD_H -DHAVE_LIBDL -DUSEAPI_DUMMY -w
 PD_JNI_CFLAGS := -Wno-int-to-pointer-cast -Wno-pointer-to-int-cast
 PD_LDLIBS := -ldl -latomic

